webdriver/tests/perform_actions/pointer.py

```diff
@@ -148,7 +148,7 @@ def test_drag_and_drop_with_draggable_element(session_new_window,
                        test_actions_page,
                        mouse_chain,
                        drag_duration):
-    new_session = session_with_new_window
+    new_session = session_new_window
```

Comment:
Why this change?

Reply:
This is from my other fix https://github.com/web-platform-tests/wpt/pull/25365, I merged it. I do not know how to rebase this PR on top of it.

Reply:
You want to pull the latest changes from the upstream repository and then run a `git rebase master` while having your branch for wheel input checked out.